tulflow/solr_api_utils.py

```diff
@@ -66,8 +66,8 @@ def delete_collection(self, collection):
         else:
             logging.info(f"Collection {collection} did not exist. Continuing without error")
 
-    def create_collection(self, collection, configset="_default" ,numShards=1, replicationFactor=1, maxShardsPerNode=1 ):
-        path = f"/solr/admin/collections?action=CREATE&name={collection}&collection.configSet=configset&numShards=1&replicationFactor=1&maxShardsPerNode=1"
```

Comment:
do we want replicationFactor hardcoded?

Reply:
ugh, no

Reply:
Do we? Seems like we might want to be flexible, right?

Reply:
yeah, thought so, but saw you changed it here to being hard-coded, hence asking

Reply:
there's a shit ton of linting / syntax errors, so i'm just gonna push that up to this branch while chcking the replication var